### PR TITLE
Accessibility Main Features

### PR DESCRIPTION
**Pull request by akrasyuk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -150,3 +150,4 @@ bld/
 
 # OpenCover
 *_coverage.xml
+/ReactWindows/Playground.Net46/.idea
```
Comment (akrasyuk) [inline .gitignore]: WebStorm files ignore

File: ReactWindows/ReactNative.Net46/Views/TextInput/ReactTextInputManager.cs [hunk 2]
```diff
@@ -273,17 +273,6 @@ public void SetIsTabStop(ReactTextBox view, bool isTabStop)
             view.IsTabStop = isTabStop;
         }
 
-        /// <summary>
```
Comment (pre10der89) [inline ReactWindows/ReactNative.Net46/Views/TextInput/ReactTextInputManager.cs]: Why is this being removed?   I assume this is based on RNW #57, which should have a lot of support for keyboard navigation?

Comment (pre10der89) [reply ReactWindows/ReactNative.Net46/Views/TextInput/ReactTextInputManager.cs]: How will you dictate the tabbing order?

File: ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs [hunk 3]
```diff
@@ -120,6 +120,28 @@ public void SetAccessibilityLabel(TFrameworkElement view, string label)
             AutomationProperties.SetName(view, label ?? "");
         }
 
+        /// <summary>
+        /// Sets the tab index property.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="tabIndex">The tab index.</param>
+        [ReactProp("tabIndex")]
+        public void SetTabIndex(TFrameworkElement view, int tabIndex)
```
Comment (pre10der89) [inline ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs]: okay, ignore my below comments about the SetTabIndex being removed, I see it is reimlemented here.

File: ReactWindows/Playground.Net46/index.windows.js [hunk 4]
```diff
@@ -7,13 +7,17 @@ import React, { Component } from 'react';
 import {
   AppRegistry,
   StyleSheet,
-  View
+  View,
```
Comment (pre10der89) [inline ReactWindows/Playground.Net46/index.windows.js]: I'm not quite sure the right approach is to change the playground from its minimal implementation.   The purpose of the playground is to be used to test an idea and then move that idea elsewhere... I'd much rather see a new project that demonstrates all Accessibility features and techniques...

File: ReactWindows/ReactNative.Net46/Views/Text/ReactTextViewManager.cs [hunk 5]
```diff
@@ -80,7 +80,7 @@ public void SetTextDecorationLine(TextBlock view, string textDecorationLineValue
         [ReactProp("selectable")]
         public void SetSelectable(TextBlock view, bool selectable)
         {
-            throw new NotImplementedException();
+            view.Focusable = selectable;
```
Comment (pre10der89) [inline ReactWindows/ReactNative.Net46/Views/Text/ReactTextViewManager.cs]: Will we need to control logical focus and keyboard focus?

File: ReactWindows/ReactNative.Net46/Views/TextInput/ReactTextInputManager.cs (hunk 2 above)
Comment (pre10der89) [reply ReactWindows/ReactNative.Net46/Views/TextInput/ReactTextInputManager.cs]: ignore I see its been moved above.

**Comment by pre10der89** [comment]
@abodalevsky @akrasyuk  I'd like to see the Playground.Net46 returned to its original form.   That project is meant to be a simple basic implementation that allows you to temporarily experiment a new idea or test a component.   Changes to it shouldn't be committed... I'd much rather see a new Accessbility focused playground that demonstrates all applicable accessbility features/techniques/concepts.

File: .gitignore (hunk 1 above)
Comment (abodalevsky) [reply .gitignore]: you may add just `.idea` it excludes it from all folders

**Pull request merged**